README.md

```diff
@@ -59,7 +59,9 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+In line 1 we create the variable weather, which is still undefined. Weather now has a value of undefined.
+In line 2 we give the variable a value of "sunny". The variable weather is now now sunny.
+In line three we are asking if the weather is in fact sunny, and based on the the previous two lines this is true. Therefore, the value returned is true.
```

Comment:
All of this is true; however, the question was specifically asking what each of those three lines evaluates to. 

Line 1, `var weather;`, evaluates to `undefined`.
Line 2, `weather = "sunny"`, evaluates to `"sunny"` because the `=` operator always evaluates to the value that it's assigning.
Line 3, `weather === "sunny"`, evaluates to true.